README.md

```diff
@@ -27,16 +29,15 @@ The primary function of the bot is to establish a 1:1 linkage between a Discord
 3. Create a webhook named "Hubs" in the channels you want it to run in. It will use this webhook to bridge chat and
    send Hubs status updates.
 
-4. Try out the bot! Type `!hubs` in a channel the bot is in to see some things you can do. The bot will bind a Discord channel and a room if it sees the room URL in the topic of the channel.
+4. Try out the bot! Type `!hubs` in a channel the bot is in to see some things you can do. The bot will bridge a Discord channel and a room if it sees the room URL in the topic of the channel.
```

Comment:
This isn't true anymore, is it? The bot only bridges rooms if they were created using the bot. (I recognize that when it restarts it uses the topic, but that's a bug we are hoping nobody notices.)

Reply:
Never mind, I guess it still is true. I didn't understand the new design.